src/Controller/Component/CrudComponent.php

```diff
@@ -547,7 +547,7 @@ public function defaults($type, $name, $config = null)
                 $this->config(sprintf('%s.%s', $type, $realName), $config);
             }
 
-            return;
+            return null;
```

Comment:
It makes it the same quality the core code is. And yes, doc block should also be correct, which it is, right?